src/Localization/Localization/test/Model.cs

```diff
@@ -0,0 +1,8 @@
+// This namespace for test resources with alternative RootNamespace
```

Comment:
This file needs to be moved into the new folder I created to separate the tests which mess with RootNamespace from those which do not.

Reply:
What if:
- Create `test-assets`
- Convert the `LocalizationTest` to ClassLibrary
- Add the test as a part of `Microsoft.Extensions.Localization.Tests`

This quite similar to what we did before into AspNetCore repo

Reply:
The test isn't of loading resources from a class-library, Something like what you suggest probably makes sense if we have more (and more complicated tests) for this kind of Localization thing in the future, but we only have the one, and it can be fixed by moving this file into `src/Localization/Localization/test/Microsoft.Extensions.Localization.Tests`.

Reply:
Ok, what I need to do now is moving the `model` class into `Microsoft.Extensions.Localization.Tests`. Am I right?

Reply:
Yes